extensions/all-extensions/pom.xml

```diff
@@ -46,6 +65,26 @@
                     <groupId>com.jcraft</groupId>
                     <artifactId>jsch</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>javax.inject</groupId>
+                    <artifactId>javax.inject</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>commons-io</groupId>
+                    <artifactId>commons-io</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>commons-lang</groupId>
+                    <artifactId>commons-lang</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>commons-collections</groupId>
+                    <artifactId>commons-collections</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.google.guava</groupId>
```

Comment:
Ok, I'll remove the import and use Supplier from JDK.

On Thu, Jun 2, 2016 at 3:11 PM, Mariano Gonzalez <notifications@github.com>
wrote:

> In extensions/all-extensions/pom.xml
> <https://github.com/mulesoft/mule/pull/2743#discussion_r65590151>:
>
> > +                    <artifactId>javax.inject</artifactId>
> > +                </exclusion>
> > +                <exclusion>
> > +                    <groupId>commons-io</groupId>
> > +                    <artifactId>commons-io</artifactId>
> > +                </exclusion>
> > +                <exclusion>
> > +                    <groupId>commons-lang</groupId>
> > +                    <artifactId>commons-lang</artifactId>
> > +                </exclusion>
> > +                <exclusion>
> > +                    <groupId>commons-collections</groupId>
> > +                    <artifactId>commons-collections</artifactId>
> > +                </exclusion>
> > +                <exclusion>
> > +                    <groupId>com.google.guava</groupId>
>
> that's a bug we need to fix. For sure I mean java.util.function.Supplier
> but screwed up with the import
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mulesoft/mule/pull/2743/files/c98fb624efa9b9914f2b46038515d8dd47bd2159#r65590151>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG96KS03xd6IJVbHF_i_bJe-H_mvf0IZks5qHxzZgaJpZM4Isq0Y>
> .
>



-- 
<http://www.mulesoft.com/>
Guillermo Fernandes, Senior Software Engineer

Av. Corrientes 316 Entre Piso, Buenos Aires, C1043AAQ, Argentina


<https://www.linkedin.com/company/www.mulesoft.com>
<https://twitter.com/mulesoft>   <https://www.facebook.com/MuleSoft>
<https://www.youtube.com/user/mulesoftvids> We're hiring!
<http://www.mulesoft.com/careers>